packager/react-packager/src/JSTransformer/worker.js

```diff
@@ -17,12 +17,6 @@ var Transforms = require('../transforms');
 // Javascript code
 function internalTransforms(sourceCode, filename, options) {
   var plugins = resolvePlugins(Transforms.getAll(options));
-  if (plugins.length === 0) {
```

Comment:
why remove this? if there're no plugins to run would be great skipping transforming the file :)